x-pack/plugins/infra/kibana.json

```diff
@@ -16,7 +16,8 @@
     "observability",
     "ruleRegistry",
     "unifiedSearch",
-    "lens"
+    "lens",
+    "guidedOnboarding"
```

Comment:
Could you help me understand why this would have to become a dependency of the `infra` plugin?

Reply:
Happy path, the EuiTour steps render on the Overview page. However, if a user navigates away via the left nav during the tour, the tour steps should still render (e.g., to the “Stream” page). The tour logic lives within the shared Observability page template, and is dependent on the `guidedOnboarding` plugin.

Reply:
I see, thanks for the explanation. But wouldn't the existing dependency on the `observability` plugin (through which the `infra` plugin renders the observability page template in the first place) already extend to the `guidedOnboarding` dependency transitively?

Reply:
@weltenwort I went back and looked through the code, and I was able to inject it as a dependency when `createLazyObservabilityPageTemplate` is called. This should remove the need to make it a dependency of each plugin. Thanks for calling this out!